pkgs/servers/homepage-dashboard/default.nix

```diff
@@ -99,12 +86,16 @@ buildNpmPackage rec {
 
   doDist = false;
 
-  passthru.tests = {
-    inherit (nixosTests) homepage-dashboard;
+  passthru = {
+    tests = {
+      inherit (nixosTests) homepage-dashboard;
+    };
+    updateScript = nix-update-script { };
   };
 
   meta = {
     description = "A highly customisable dashboard with Docker and service API integrations.";
```

Comment:
sorry not related to this PR, but I just noticed that description must not end in a period 

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

Reply:
Good catch, I've added another commit to fix this. Thank you!